Check the  returned effect after setting selected objects for a task and
reload the option descriptors if necessary.

--- evms-2.5.5/ui/cli/interpreter.c	2005-06-01 10:20:32.000000000 -0500
+++ evms-2.5.5-fix/ui/cli/interpreter.c	2006-09-01 11:54:50.903202365 -0500
@@ -2400,6 +2400,13 @@
             Error = EINVAL;
 
         }
+        else
+        {
+          if (Task_Result & EVMS_Effect_Reload_Options) {
+            Clear_Option_Descriptors();
+            Get_Option_Descriptors();
+          }
+        }
 
       }
       else
